website/sidebars.js

```diff
@@ -926,6 +926,8 @@ const sidebarSettings = {
           items: [
             "guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project",
             "guides/dbt-ecosystem/databricks-guides/dbt-unity-catalog-best-practices",
+            "guides/dbt-ecosystem/databricks-guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs",
```

Comment:
hey @dbt-markwan , this is awesome, and glad you added it to the sidebar! do you think this guide belongs in the [orchestration](https://github.com/dbt-labs/docs.getdbt.com/blob/2a4aca437ffe3449ecd555f7f5c9f14e8d680b75/website/sidebars.js#L812) section  since it's about orchestrating dbt cloud.  ? I'd like to double check with @amychen1776 to ensure if this guide fits under the dbt ecosystem -> databricks and dbt folder as well 🙏

Reply:
thanks so much for your input @amychen1776 ! ✨